langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAiAudioTranscriptionModelIT.java

```
@@ -0,0 +1,108 @@
+import dev.langchain4j.data.audio.Audio;
+import dev.langchain4j.model.audio.AudioTranscriptionRequest;
+import dev.langchain4j.model.audio.AudioTranscriptionResponse;
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Integration test for the Azure OpenAI audio transcription model.
+ */
+public class AzureOpenAiAudioTranscriptionModelIT {
+
+    private static final Logger logger = LoggerFactory.getLogger(AzureOpenAiAudioTranscriptionModelIT.class);
+
+    @Test
+    void should_transcribe_audio_with_new_api() throws IOException {
+        String endpoint = System.getenv("AZURE_OPENAI_ENDPOINT");
+        String apiKey = System.getenv("AZURE_OPENAI_KEY");
+        String deploymentName = System.getenv("AZURE_OPENAI_AUDIO_DEPLOYMENT_NAME");
+
+        // Skip test if environment variables aren't configured
+        Assumptions.assumeTrue(
+                endpoint != null && !endpoint.isBlank(), "AZURE_OPENAI_ENDPOINT environment variable not set");
+        Assumptions.assumeTrue(apiKey != null && !apiKey.isBlank(), "AZURE_OPENAI_KEY environment variable not set");
+        Assumptions.assumeTrue(
+                deploymentName != null && !deploymentName.isBlank(),
+                "AZURE_OPENAI_AUDIO_DEPLOYMENT_NAME environment variable not set");
+
+        AzureOpenAiAudioTranscriptionModel model = AzureOpenAiAudioTranscriptionModel.builder()
+                .endpoint(endpoint)
+                .apiKey(apiKey)
+                .deploymentName(deploymentName)
+                .responseFormat(AudioTranscriptionFormat.JSON)
+                .logRequestsAndResponses(true)
+                .build();
+
+        File audioFile = new File("src/test/resources/hello.mp3");
+        Assumptions.assumeTrue(audioFile.exists(), "Test audio file not found: src/test/resources/hello.mp3");
```

If there is no file, the test should fail instead of being ignored